app/views/users/_edit.haml

```diff
@@ -122,61 +122,180 @@
       .row
         .col-md-12
           %h3
-            = t(".receive_email_notifications")
```

Comment:
Should we clean up existing translations for this string? Or it will be done automatically when we sync with the translation platform?

Reply:
You only need to update the English translations, other languages are managed by webtranslateit.